java/src/org/dartlang/vm/service/RemoteServiceRunner.java

```diff
@@ -21,9 +21,8 @@
 public interface RemoteServiceRunner {
```

Comment:
Passing the `serviceName` into `assServiceRunner` inforces the static nature of the value.
A getter can potentially return different values.
Another reason is to keep the same approach used for the Completer interface.